default/python/universe_generation/names.py

```diff
@@ -3,7 +3,6 @@
 
 import freeorion as fo
 
-
```

Comment:
Why remove this line? I thought there should be two blank lines between the imports section and the subsequent code sections?

Reply:
Lets it be 2, I just updated description with settings screenshot.